specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/examples/WorkbookAdd.json

```diff
@@ -0,0 +1,73 @@
+    "workbookProperties": {
+      "location": "west us",
+      "kind": "shared",
+      "properties": {
+        "displayName": "tttt",
+        "serializedData": "{\"version\":\"Notebook/1.0\",\"items\":[{\"type\":1,\"content\":\"{\"json\":\"## New workbook\\r\\n---\\r\\n\\r\\nWelcome to your new workbook.  This area will display text formatted as markdown.\\r\\n\\r\\n\\r\\nWe've included a basic analytics query to get you started. Use the `Edit` button below each section to configure it or add more sections.\"}\",\"halfWidth\":null,\"conditionalVisibility\":null},{\"type\":3,\"content\":\"{\"version\":\"KqlItem/1.0\",\"query\":\"union withsource=TableName *\\n| summarize Count=count() by TableName\\n| render barchart\",\"showQuery\":false,\"size\":1,\"aggregation\":0,\"showAnnotations\":false}\",\"halfWidth\":null,\"conditionalVisibility\":null}],\"isLocked\":false}",
+        "category": "workbook",
+        "description": "Sample workbook"
+      },
+      "tags": {
+        "TagSample01": "sample01",
+        "TagSample02": "sample02"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/6b643656-33eb-422f-aee8-3ac145d124af/resourcegroups/my-resource-group/providers/microsoft.insights/workbooks/deadb33f-8bee-4d3b-a059-9be8dac93960",
+        "type": "microsoft.insights/workbooks",
+        "location": "westus",
+        "name": "deadb33f-8bee-4d3b-a059-9be8dac93960",
+        "kind": "shared",
+        "properties": {
+          "displayName": "Blah Blah Blah",
+          "userId": "userId",
+          "serializedData": "{\"version\":\"Notebook/1.0\",\"items\":[{\"type\":1,\"content\":\"{\"json\":\"## New workbook\\r\\n---\\r\\n\\r\\nWelcome to your new workbook.  This area will display text formatted as markdown.\\r\\n\\r\\n\\r\\nWe've included a basic analytics query to get you started. Use the `Edit` button below each section to configure it or add more sections.\"}\",\"halfWidth\":null,\"conditionalVisibility\":null},{\"type\":3,\"content\":\"{\"version\":\"KqlItem/1.0\",\"query\":\"union withsource=TableName *\\n| summarize Count=count() by TableName\\n| render barchart\",\"showQuery\":false,\"size\":1,\"aggregation\":0,\"showAnnotations\":false}\",\"halfWidth\":null,\"conditionalVisibility\":null}],\"isLocked\":false}",
+          "version": "Notebook/1.0",
+          "category": "workbook",
+          "timeModified": "2021-08-24T06:56:31.6301521Z",
+          "storageUri": null,
+          "description": "Sample workbook",
+          "revision": "1e2f8435b98248febee70c64ac22e1bb"
+        },
+        "tags": {
+          "TagSample01": "sample01",
+          "TagSample02": "sample02"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/6b643656-33eb-422f-aee8-3ac145d124af/resourcegroups/my-resource-group/providers/microsoft.insights/workbooks/deadb33f-8bee-4d3b-a059-9be8dac93960",
+        "type": "microsoft.insights/workbooks",
```

Comment:
nit, shouldn't it be capitalized in the response as 'Microsoft.Insights/workbooks' ?